src/data/requirements/camo/digital.js

```diff
@@ -0,0 +1,122 @@
+export default {
```

Comment:
We should probably separate this into `digital_1` and `ditigal_2` files (as in game) to properly separate the two "Jungle Digital" duplicates. then we won't run into any issues when calling it. Same with "Woodland" category. But I can fix that later! 😄

Reply:
Yeah I did think about this, but I actively decided to keep it as 1 category because it seemed simpler? I'm happy to split them up if you prefer, but I think it's just a simplicity thing, we can keep the duplicate without having any real issues.